lib/foreman_tasks/dynflow/persistence.rb

```diff
@@ -11,7 +11,8 @@ def save_execution_plan(execution_plan_id, value)
         begin
           on_execution_plan_save(execution_plan_id, value)
```

Comment:
Here, at this point, I would like to be able to access properties from the job, such as name, comments, etc.. to actually save ActiveJob jobs with a proper name in the Tasks table. I was not able to do this, and I'm not sure if it's possible without changing Dynflow cc @iNecas 

My idea was to have each job provide an arbitrary hash with things like name, a comment explaining what the job does, possible debugging steps, etcetera, but I cannot pick random stuff from the job here.

If such an interface existed, we would be able to show nicer names on the /tasks table than the class name, e.g: "Pulp::Repository::Sync" could show up as "Pulp: Syncing repository EPEL", and a comment explaining what the task does, etc..